modules/core/src/lib/mode-handlers/draw-circle-by-bounding-box-handler.js

```diff
@@ -17,10 +17,19 @@ export class DrawCircleByBoundingBoxHandler extends TwoClickPolygonHandler {
       return result;
     }
 
+    const modeConfig = this.getModeConfig();
+    let { options = {} } = modeConfig;
+    if (options.steps < 16) {
+      console.warn(`Minimum steps to draw a circle is 16 `); // eslint-disable-line no-console,no-undef
```

Comment:
Is 16 the minimum in turf? If not, then we should make the minimum in nebula match turf. You can do further validation in your app if you want to make 16 the minimum.

Reply:
4 is the minimum step required in turfjs. If set to 4, it will draw a rhombus. To look more like a circle configured it to 16.

Sure will update the changes